settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
Yes, the breaking is related to the Gradle version check, which is caused by the `demo-assethelper` configuration. The thing is, building the plugin should not cause any demo projects to be rebuilt. 
We need to be able to test against different Gradle versions, independently of the version used to build the plugin.

See also https://github.com/novoda/sqlite-analyzer/pull/19